docs/blocks-scaffolding.md

```diff
@@ -0,0 +1,45 @@
+# Scaffolding blocks
```

Comment:
This title might not be immediately evident to folks.

Reply:
`Speeding up writing blocks`  - what about this title?

Reply:
I would go with "Generate Blocks with WP-CLI"

Reply:
Will update 👍